apps/voting/src/shared/election-server.ts

```diff
@@ -71,6 +70,52 @@ function reviveBallotSubmission(value: DatabaseBallotSubmissionSerializable): Da
     };
 }
 
+type DelegationSerializable = {
+    /** The index of the delegation entry in the database */
+    id: bigint;
+    /** The delegatee account address */
+    toAccount: Base58String;
+    /** The delegator account address */
+    fromAccount: Base58String;
+    /** The transaction hash corresponding to the submission */
+    transactionHash: HexString;
+    /** The slot time of the block the submission is included in */
+    blockTime: string;
+    /** Whether the ballot could be verified */
```

Comment:
This comment does not seem accurate. Presumably the weight is the weight that was delegated?

Reply:
Good catch!